### DISCUSSION TITLE
Check mental condition

### DISCUSSION THREAD

**Discussion by way3-3** [way3-3/EmployeeManagementSystem#10] [Polls]
@arefinthecybersec @shehabrakib @for41 @Seyam69 @Thor-unleash @fix3-3

**Comment by arefinthecybersec** [comment]
vai tmi tuh dki poll banay rkco bndhu

**Reply by way3-3** [reply]
haa vai tomader binodoner jonno korsilam.Dekhla o na tokhon etodin por dekhtoso😁😁
